internal/locate/region_cache.go

```diff
@@ -3252,6 +3252,21 @@ func (s *Store) GetPeerAddr() string {
 	return s.peerAddr
 }
 
+func (s *Store) updateServerLoadStats(estimatedWaitMs uint32) {
+	if estimatedWaitMs != 0 {
+		estimatedWait := time.Duration(estimatedWaitMs) * time.Millisecond
+		// Update the estimated wait time of the store.
+		loadStats := &storeLoadStats{
+			estimatedWait:     estimatedWait,
+			waitTimeUpdatedAt: time.Now(),
+		}
+		s.loadStats.Store(loadStats)
+	} else {
+		// Mark the server is busy (the next incoming READs could be redirect to expected followers.)
+		s.healthStatus.markAlreadySlow()
```

Comment:
Why putting the `markAlreadySlow` in the `updateServerLoadStats`? I think it shoud remain in the `OnServerIsBusy` action function.

Reply:
done.